File: CCTI.Appl.MainForm/LoginForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CCTI.Appl.MainForm
{
    public partial class LoginForm : Form
    {
        public LoginForm()
        {
            InitializeComponent();
        }

        private void label1_Click(object sender, EventArgs e)
        {

        }

        private void uxCancel_Click(object sender, EventArgs e)
        {
            uxUserName.Clear();
            uxPassword.Clear();
            uxUserName.Focus();
        }

        private void uxLogin_Click(object sender, EventArgs e)
        {
            if (uxUserName.Text == "ADMIN" && uxPassword.Text == "CPRG254")
            {
                PresentationForm prstationFrm = new PresentationForm();
                prstationFrm.Show();
            }
            else
            {
                MessageBox.Show("Wrong Password or User Name. Try Again");
                uxUserName.Clear();
                uxPassword.Clear();
                uxUserName.Focus();
            }
               

        }

        private void LoginForm_Load(object sender, EventArgs e)
        {

        }
    }
}

File: CCTI.Appl.StartForms/LoginForm.vb
Public Class LoginForm
    Private prstationFrm As PresentationForm

    Private Sub uxLogin_Click(sender As Object, e As EventArgs) Handles uxLogin.Click
        If (uxUserName.Text = "ADMIN" And uxPassword.Text = "CPRG254") Then
            'Me.Hide()
            If prstationFrm Is Nothing OrElse prstationFrm.IsDisposed Then
                prstationFrm = New PresentationForm()
            End If
            prstationFrm.Show()
            prstationFrm.BringToFront()


        Else

            MessageBox.Show("Wrong Password or User Name. Try Again")
            uxUserName.Clear()
            uxPassword.Clear()
            uxUserName.Focus()
        End If
    End Sub

    Private Sub uxCancel_Click(sender As Object, e As EventArgs) Handles uxCancel.Click
        uxUserName.Clear()
        uxPassword.Clear()
        uxUserName.Focus()
    End Sub

    Private Sub LoginForm_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub

    Private Sub uxClose_Click(sender As Object, e As EventArgs) Handles uxClose.Click
        Close()
    End Sub
End Class
